examples/medplum-provider/src/components/encountertasks/TaskQuestionnaireForm.tsx

```diff
@@ -16,6 +16,7 @@ export const TaskQuestionnaireForm = ({ task, onChangeResponse }: TaskQuestionna
   const [questionnaire, setQuestionnaire] = useState<Questionnaire | undefined>(undefined);
   const [questionnaireResponse, setQuestionnaireResponse] = useState<QuestionnaireResponse | undefined>(undefined);
   const [loading, setLoading] = useState(false);
+  const [notFound, setNotFound] = useState(false);
```

Comment:
The model we have used elsewhere is to save the `OperationOutcome`, which will often have more details.  There are other errors than can happen other than "Not found".

```suggestion
  const [outcome, setOutcome] = useState(undefined);
```